frontend/cvat/src/app/share-browse-modal/share-browse-modal.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Is this a component used for testing? Can it be removed?

Reply:
Removed it